### PR TITLE
Fix code block

### PR DESCRIPTION
**Pull request by juliankrispel**

### CODE REVIEW

**Review by mxstbr — APPROVED** [review]
Let's try this!

File: src/modifiers/handleNewCodeBlock.js [hunk 1]
```diff
@@ -20,8 +20,18 @@ const handleNewCodeBlock = editorState => {
...
+      "",
+      data
+    );
+    return splitBlockAndChange(
+      editorStateWithCodeBlock,
+      undefined,
+      undefined,
+      false
+    );
   }
```
Comment (juliankrispel) [inline src/modifiers/handleNewCodeBlock.js:35]: this is just prettier doing its thing

Comment (mxstbr) [reply src/modifiers/handleNewCodeBlock.js:35]: Yeah we should just add that as a precommit hook, honestly

**Comment by mxstbr** [comment]
Fixes the issue properly :tada:

**Pull request merged**